tests/keras/test_keras_model_export.py

```diff
@@ -438,6 +438,48 @@ def test_model_save_accepts_conda_env_as_dict(model, model_path):
     assert saved_conda_env_parsed == conda_env
 
 
+@pytest.mark.large
+def test_model_save_persists_requirements_in_mlflow_model_directory(
+    model, model_path, keras_custom_env
+):
+    mlflow.keras.save_model(keras_model=model, path=model_path, conda_env=keras_custom_env)
+
+    # pyfunc_conf = _get_flavor_configuration(model_path=model_path, flavor_name=pyfunc.FLAVOR_NAME)
+    saved_pip_req_path = os.path.join(model_path, "requirements.txt")
+    assert os.path.exists(saved_pip_req_path)
+
+    with open(keras_custom_env, "r") as f:
+        keras_custom_env_parsed = yaml.safe_load(f)
+    with open(saved_pip_req_path, "r") as f:
+        requirements = f.read().split("\n")
+
+    assert keras_custom_env_parsed["dependencies"][-1]["pip"] == requirements
```

Comment:
These code are similar in every suites. Can we create a helper method, to reduce duplicated code ?